### ISSUE TITLE
Can you post a validation loss?

### ISSUE THREAD

**Issue by ildoonet** [michalfaber/keras_Realtime_Multi-Person_Pose_Estimation#45]
I saw training loss around 20~30 values in your readme.

What about validation loss value?

Is validation loss value similar to the training one?

**Comment by anatolix** [comment]
It is not everything easy with losses in this project.
1. training and validation loss isn't comparable with caffe model because we use pictures for validation which caffe model use for training.
2. current projects hdf5 select persons for training differently which make dataset easier and makes different scores. see this for details
https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/issues/41

This loss I have for this project:
[cpp.csv.zip](https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/files/1598048/cpp.csv.zip)
This is for my python augmentation fork
[python.csv.zip](https://github.com/michalfaber/keras_Realtime_Multi-Person_Pose_Estimation/files/1598050/python.csv.zip)
As I said above they are incomparable.

May be best idea to use COCO metric committed in my fork
